### ISSUE TITLE
Debowerify breaks if given no options?

### ISSUE THREAD

**Issue by ashrewdmint** [eugeneware/debowerify#63]
When trying to run `$ browserify -t debowerify path/to/files/*.js -o path/to/output.js` I get this:

```
/.../node_modules/debowerify/index.js:79
    var bowerOptions = options.bowerOptions || {};
                              ^
TypeError: Cannot read property 'bowerOptions' of undefined
```

If I make these changes to `debowerify`'s `index.js`, it works:

```
8 | module.exports = function (file, options) {
9 |   if (!options) options = {};
```

I have upgraded to the latest version of `node` and `npm` but it still doesn't seem to work as-is.

**Comment by sukima** [comment]
Have you considered sending the change you made as a pull request?

**Comment by ashrewdmint** [comment]
OK, here is a pull request with the change mentioned above.

**Comment by eugeneware** [comment]
Should be fixed by #64. Thanks!
